### DISCUSSION TITLE
[BUG] Bad markdown causing `parseList Unable to parse list...` shouldn't silently fail

### DISCUSSION THREAD

**Discussion by darthwalsh** [vslinko/obsidian-outliner#466] [Ideas]
**Describe the bug**
I had a problem with bad spaces/tabs in my list, causing command "Move list and sublists up" to just silently fail, with no error message.

**To Reproduce**
Create file with mixed tabs, spaces:

```
- [ ] A
- [ ] B
 - [ ] Space Indent
	 - [ ] Tab Indent
```

Put cursor on line `B`

CMD + SHIFT + UP to execute command "Move list and sublists up"

**Expected behavior**
If unable to parse list because of unexpected indent, please show an error message.

**Screenshots**
![image](https://github.com/vslinko/obsidian-outliner/assets/2829438/0b63eef1-4281-43a4-89ae-f45c024dc273)

(Or, see #305 or #352)

**Environment (please complete the following information):**
 - OS: macOS Desktop
 - Obsidian Version: v1.3.5
 - Plugin Version: 4.7.1

**Additional context**
This was very confusing! I had a big outliner file, and couldn't figure out why Outliner commands were failing to run.

Eventually I figured out how to open devtools with Outliner debug logging, and googled the issue.

It would have been great if outliner could show an error popup `Unable to parse list: expected indent "S", got "T"` -- even better if outliner could highlight which line was causing the parse problem?

**Comment by csss1234** [comment]
Solution https://github.com/vslinko/obsidian-outliner/discussions/485#discussion-5691533
